### DISCUSSION TITLE
Ignore Case and smart completions

### DISCUSSION THREAD

**Discussion by semi710** [ajeetdsouza/zoxide#535] [Q&A] [answered]
How can i make zoxide go in dir `Downloads` if I type `downloads`?

and also I have observed that

if my dir structure is `This/is/A/very/big/path`
and I do `z path` it works
but if I have similar dir structure I would like z to work with `z ver/bi/path`

where I am kinda giving the dir structure and not typing the full path.

**Comment by ajeetdsouza** [comment, marked as answer]
> How can i make zoxide go in dir `Downloads` if I type `downloads`?

zoxide already does this. If you type `z downloads`, it should take you to the `Downloads` directory.





> but if I have similar dir structure I would like z to work with `z ver/bi/path`

Use spaces instead of `/`s. `z ver bi path` will take you to the correct directory.
